### ISSUE TITLE
Create an SAP Cloud Platform ABAP Environment Trial User

### ISSUE THREAD

**Issue by rvinod14** [SAPDocuments/Tutorials#5223]
Hello Experts,
Tutorial URL: https://developers.sap.com/tutorials/abap-environment-trial-onboarding.html

On Step 6 - After importing the service key, I get error "Please ensure that you have a user for the service instance."
<img width="428" alt="Message" src="https://user-images.githubusercontent.com/57740183/83123570-b48eab00-a0f2-11ea-825d-d596787898ba.png">

Regards,
Vinod Iyer

**Comment by mervey45** [comment]
Hi @rvinod14 ,

please make sure you have the latest [ADT installation:](https://tools.hana.ondemand.com/#abap)
To install the front-end component of ADT, proceed as follows:

Get an installation of Eclipse 2020-03 (e.g. Eclipse IDE for Java Developers)
In Eclipse, choose in the menu bar Help > Install New Software...
Enter the URL https://tools.hana.ondemand.com/latest
Press Enter to display the available features.
Select ABAP Development Tools and choose Next.
On the next wizard page, you get an overview of the features to be installed. Choose Next.
Confirm the license agreements and choose Finish to start the installation.


Please also make sure, that your proxy settings are proper: 

**ADT -> Window -> Preferences ->General -> Network Connections** 

![image](https://user-images.githubusercontent.com/27953324/83125299-a54e3400-a0d7-11ea-9b31-0f380feb43e7.png)


Thank you.

Best regards, 
Merve

**Comment by rvinod14** [comment]
Hi Merve,

I have installed Eclipse 2020-03 & also the ABAP Development Tools from site https://tools.hana.ondemand.com/latest.
The proxy settings are per the screenshot provided by you.
When I try to stop the loading of page and click on Back. A screen comes showing the Service Instance Settings. Here Email is displayed, but not the User ID and SAP System ID.
I had also tried to delete the scp trial account and create new one.

Regards,
Vinod Iyer
![image](https://user-images.githubusercontent.com/57740183/83126414-64b1e300-a0f6-11ea-8ae4-fcf7ec0c382a.png)

<img width="301" alt="Service Instance Settings" src="https://user-images.githubusercontent.com/57740183/83127888-5238a900-a0f8-11ea-9b4a-9f74375e31c1.png">

**Comment by mervey45** [comment]
Hi @rvinod14 ,

please delete your cache and try to logon with your browser then instead.

![image](https://user-images.githubusercontent.com/27953324/83129799-a08c7e80-a0dd-11ea-94f5-ca916cde1c3c.png)

After trying in your credentials wait a few seconds and then it should work.

You also have to make sure, that your e-mail address, you've registered in the trial cockpit must be the same e-mail address you have entered, when you created your abap trial user.

Best regards, 
Merve

**Comment by rvinod14** [comment]
Hi Merve,

I tried to login from browser, but still its the same error.
I have mentioned the same e-mail address when creating abap trial user.
I noticed that after importing the service key, logon page is not coming in fact its automatically getting authenticated.
I'm attaching the  steps.
[New ABAP Cloud Project.docx](https://github.com/SAPDocuments/Tutorials/files/4695088/New.ABAP.Cloud.Project.docx)

Regards,
Vinod Iyer
